src/ProductBundle/Controller/ProductVariationAdminController.php

```diff
@@ -36,8 +41,15 @@ public function createAction(Request $request = null)
             throw new AccessDeniedException();
         }
 
+        // NEXT_MAJOR: Keep FQCN when bumping Symfony requirement to 2.8+.
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
```

Comment:
Please use the `::class` notation as much as you can.

Reply:
Is it better now?